src/framework/components/collision/trigger.js

```diff
@@ -46,14 +46,21 @@ Object.assign(pc, function () {
 
                 var startTransform = new Ammo.btTransform();
                 startTransform.setIdentity();
-                startTransform.getOrigin().setValue(pos.x, pos.y, pos.z);
+                var origin = startTransform.getOrigin();
+                origin.setValue(pos.x, pos.y, pos.z);
                 startTransform.setRotation(ammoQuat);
 
                 var motionState = new Ammo.btDefaultMotionState(startTransform);
                 var bodyInfo = new Ammo.btRigidBodyConstructionInfo(mass, motionState, shape, localInertia);
 
+                delete Ammo.btDefaultMotionState.__cache__[motionState.ptr];
```

Comment:
This looks odd. Can you add code comments explaining the requirement for the above line please?

Reply:
Removed this bit.